UKHomeOffice/cerberus-service
# COP-11935 Added a relist button to completed tasks

## Description
When going to a completed task there is now a relist button. It will take you to a relist form where you can fill out a reason for relisting. After submitting the form you'll be given a message to say it's a success and you'll be allowed to return to the task details page. The task will now be in progress and assigned to you and have a note featuring your reason for relisting.
![image](https://user-images.githubusercontent.com/12000340/191261057-1b3d89e0-1ae4-4f72-947a-54fc8926534c.png)
![image](https://user-images.githubusercontent.com/12000340/191261084-fd1b4444-194a-421a-8bb8-0fc04143a3c8.png)
![image](https://user-images.githubusercontent.com/12000340/191261107-f98b4466-cd6c-410f-8c99-9ef62ed1d73a.png)
![image](https://user-images.githubusercontent.com/12000340/191261311-92a7d4ca-5759-4bb4-bcc6-b4beaf1cba6b.png)

## To Test
1. Go to a completed task
2. Verify the relist button appears
3. Click the relist button
4. Attempt to submit without a relist reason (you shouldn't be able to)
5. Fill out a relist reason and submit the form
6. View the message
7. Return to the task details page
8. Confirm it's in progress, assigned to you, and has a note with the new reason
9. Confirm that the relist button no longer appears but the other action buttons do
10. Go to a new and issued task and confirm the relist button isn't there either

## Developer Checklist

\* Required



- [x] Pull request URL linked to JIRA ticket? \* https://support.cop.homeoffice.gov.uk/browse/COP-11935
